cpp/src/parquet/column_writer.cc

```diff
@@ -2475,32 +2651,37 @@ std::shared_ptr<ColumnWriter> ColumnWriter::Make(ColumnChunkMetaDataBuilder* met
     encoding = properties->dictionary_index_encoding();
   }
   switch (descr->physical_type()) {
-    case Type::BOOLEAN:
+    case Type::BOOLEAN: {
+      if (bloom_filter != nullptr) {
+        throw ParquetException("Bloom filter is not supported for boolean type");
```

Comment:
Should we throw in the constructor of `TypedColumnWriterImpl<BooleanType>` instead to make here looks better?